config.toml

```
@@ -124,26 +120,11 @@ You can subscribe to our [newsletter](/contact/) or contact us at [info.sonar@re
 # Menu links that appear on the left sidebar
 
 [[menu.main]]
- 	name = "IR hosting"
- 	url  = "/iraas/"
+ 	name = "The project"
+ 	url  = "/project/"
  	weight = 10
```

```suggestion
 	weight = 20
```

My suggestion here aims at showing "The service" above "The project" in the sidebar. (see also suggestion below)